daml-lf/spec/transaction.rst

```diff
@@ -503,6 +505,11 @@ Containing the result of the exercised choice.
 New optional field `contract_key` is now set when the exercised
 contract has a contract key defined.
```

Comment:
Restriction on key contents must be mentioned here, and also under `message KeyWithMaintainers`.

Incidentally, it would be cleaner to use `message KeyWithMaintainers` instead of adding field `key_maintainers` as well, but I'll let @remyhaemmerle-da speak on that.

Reply:
Added and using `KeyWithMaintainers` now.

Reply:
The restriction still must be mentioned here, because this part does not use `KeyWithMaintainers`, even though it will in version 9.